mne/report.py

```diff
@@ -1492,19 +1500,17 @@ def _init_render(self, verbose=None):
                       'bootstrap.min.js', 'jquery-ui.min.css',
                       'bootstrap.min.css']
 
-        include = list()
         for inc_fname in inc_fnames:
             logger.info('Embedding : %s' % inc_fname)
             fname = op.join(op.dirname(__file__), 'html', inc_fname)
             with open(fname, 'rb') as fid:
                 file_content = fid.read().decode('utf-8')
             if inc_fname.endswith('.js'):
-                include.append(u'<script type="text/javascript">' +
-                               file_content + u'</script>')
+                self.include[fname] = \
+                    f'<script type="text/javascript">{file_content}</script>'
             elif inc_fname.endswith('.css'):
-                include.append(u'<style type="text/css">' +
-                               file_content + u'</style>')
-        self.include = ''.join(include)
+                self.include[fname] = \
+                    f'<style type="text/css">{file_content}</style>'
```

Comment:
small nitpick: above in `add_custom_css` there are newlines before open tag, between open tag and CSS code, and before close tag.  Same is true of `add_custom_js`.  shouldn't it be consistent here versus there?

Reply:
I understand, I don't really know, which is more appropriate for html-code, but I went for the newline solution in both thinking it may improve readability (and add_custom_js was the more recent addition). Ok?